src/app/blueprint.css

```diff
@@ -1,50 +1,57 @@
-/* Styles for root page */
-.page {
-  font-family: system-ui, "Segoe UI", Roboto, Helvetica, Arial, sans-serif,
-    "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+body {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
+    Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;
 }
 
-/* Styles for feature component */
-
 .feature {
-  background-color: #ffffff;
+  display: flex;
+	justify-content: center;
```

Comment:
I think we use different prettier configs, so the spacing looks weird in the diff. Let me try to come up with a common config to resolve this.